std_msgs/Header header
float64 x
float64 y
float64 heading
float64 imu_heading
float64 gps_heading
float64 gps_heading_under_4_headAcc
float64 gps_heading_raw_data
float64 heading_from_position
float64 dist
float64 roll
float64 pitch
float64 dr_x
float64 dr_y
float64 dead_m
int64 encoder_left
int64 encoder_right
float64 dead_right
float64 distance
float64 encoder_x
float64 encoder_y
int64 hAcc
float64 headAcc
int64 gear
float64 speeed
float64 gspeed
float64 dis
float64 position_error_between_encoder_and_gps
geometry_msgs/Quaternion orientation
float64 distance_x
float64 distance_y
float64 distance_z
